### PR TITLE
feat(demo): v0.5.0 docs + Kata Design System (KDS) page

### LINKED ISSUE #22: v0.5.0 docs + Kata Design System (KDS) page
Convert the demo into a somewhat functional KDS website + component kit documentation. See [atlaskit's homepage](https://atlaskit.atlassian.com/) for reference.

### PR DESCRIPTION
**Pull request by resir014**
Fixes #22.

### CODE REVIEW

**Comment by resir014** [comment]
After this PR is merged, we could start documenting all the components that we have, as well as adding documentations to new components.

File: website/src/modules/core/components/SidebarSubTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const SidebarSubTitle = styled('h1')`
+  margin-bottom: 1.846153846rem /* $space-3 */;
```
Comment (reyhansofian) [inline website/src/modules/core/components/SidebarSubTitle.tsx]: to keep it consistent, we should use space from `variables`

File: website/src/modules/docs/pages/page.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import DocsDashboard from '../components/DocsDashboard';
+import DocsDashboardContent from '../components/DocsDashboardContent';
+import Loading from '../../core/components/Loading';
+
+interface RouteParams {
+  page: string;
+}
+
+const DocsPage: React.SFC<RouteComponentProps<RouteParams>> = ({ match }) => {
+  console.log(match.params.page);
```
Comment (reyhansofian) [inline website/src/modules/docs/pages/page.tsx]: is it intended?

Comment (resir014) [reply website/src/modules/docs/pages/page.tsx]: Absolutely not. I forgot to remove that. 😅

**Pull request merged**